iexec-sms-library/src/main/java/com/iexec/sms/api/config/SconeServicesProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.iexec.sms.api.config;
+
+import com.iexec.common.tee.TeeEnclaveProvider;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
Do we need setters ?
Shouldn't those properties be immutable ?

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/119#discussion_r966682978 for a reply :)